File: C#/C#-e-Orientação-a-Objetos-coleções-arquivos-e-bibliotecas/trabalhando-com-arrays-e-colecoes/Conta/ContaCorrente.cs
public class ContaCorrente
{
    public Int32 Agencia { get; set; }
    public string Conta { get; set; }
    public double Saldo { get; set; }


    public ContaCorrente(Int32 agencia, string conta, double saldo)
    {
        Agencia = agencia;
        Conta = conta;
        Saldo = saldo;
    }

}

File: JAVA/ORIENTACAO-A-OBJETO/ATIVIDADES/orientacao-a-objeto/video2/atividade  4 ok/Banco/src/ContaCorrente.java
public class ContaCorrente extends Conta {
    private double LimiteChequeEspecial;

    public ContaCorrente(int numero, String titular, double saldo, double limiteChequeEspecial) {
        super(numero, titular, saldo);

        LimiteChequeEspecial = limiteChequeEspecial;
    }

    public double getLimiteChequeEspecial() {
        return LimiteChequeEspecial;
    }

    public void setLimiteChequeEspecial(double limiteChequeEspecial) {
        LimiteChequeEspecial = limiteChequeEspecial;
    }

    @Override
    public void calculoSaldo() {
        super.calculoSaldo();
    }
}
